tutorial_sarscov2.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: default
+permalink: /tutorial/sars-cov2/
+---
+
+This tutorial shows you the basic of how to interact with V-pipe.
+
+For the purose of this Tutorial, we will work with the `sars-cov2` branch which is [adapted for the SARS-CoV-2 virus]({{ "/sars-cov-2/" | relative_url }}).
+
+> **Organisation of Data** :
+>
+> V-pipe expects its samples data organized in a two-level hierarchy:
+>
+> - Input files to be grouped by samples (e.g.: patient samples or biological replicates of an experiment).
+> - A second level for distinction of datasets belonging to the same sample (e.g.: sample dates).
+> - inside, the directory `raw_data` hold the sequencing output in FASTQ format (optionally compressed with GZip)
+> - When in split files, paired-ends reads need to have `_R1` and `_R2` suffixes in their name.
+
+*[FASTQ]: [represents DNA sequencer reads along with quality scores](https://en.wikipedia.org/wiki/FASTQ_format)
+
+## Preparing a small dataset
+
+You can run the first test on your workstation or a good laptop.
+
+First you'll need to prepare the data:
+
+ - For that test, you need to download from SRA the following runs:
+   [SRR10903401](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR10903401) and
+   [SRR10903402](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR10903402)
+   
+   (hint: click on the _data_ tab)
+ - Then arrange the downloaded file to have the following structure
```

Comment:
'Arrange and rename (_R1 and _R2 suffixes) the files to have the following structure:'